app/src/components/Table/Table.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+
+const styles = theme => ({
+  cell: {
+    minWidth: 25,
+    maxWidth: 500,
+    wordWrap: 'break-word'
+  }
+})
+
+const SimpleTable = ({ rows, labels, classes }) => {
+  return (
+    <Table>
+      <TableHead>
+        <TableRow>
+          {labels.map(label => (
+            <TableCell className={classes.cell}>{label}</TableCell>
+          ))}
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {rows.map((row, index) => (
+          <TableRow key={index}>
+            <TableCell className={classes.cell}>{row.name}</TableCell>
+            <TableCell className={classes.cell}><code>{row.solution}</code></TableCell>
```

Comment:
Since you want to go with a generic table component here, I would name the keys in `row` as `key` and `value` respectively.

Reply:
Assuming you have only two values. I think is better to create a SolutionTable component.

Reply:
as we have formatting (<code>) specifically for solutions

Reply:
Sure.